rewrite/main.rs

```diff
@@ -133,11 +142,19 @@ fn main() {
             // TODO: support multiple UDF files
             let codegen_object =
                 codegen_simulator::CodeGenSimulator::generate_code_blocks(visitor_results, udfs);
-            write_to_handlebars(
-                &codegen_object,
-                bin_dir.join("filter.rs.handlebars"),
-                PathBuf::from(matches.value_of("output").unwrap()),
-            );
+            if distributed.is_none() {
+                write_to_handlebars(
+                    &codegen_object,
+                    bin_dir.join("filter.rs.handlebars"),
```

Comment:
Instead of having this large if-statement, just change the name, since that is apparently all that is necessary.

Reply:
What do you mean?  Put it as an argument to write_to_handlebars and then do a similar if statement inside?

Reply:
"distributed_filter.rs.handlebars" vs "filter.rs.handlebars"

Reply:
The rest of the code looks the same to me.

Reply:
We want to support both at once though.  It should be an argument to main.

Reply:
```rust
let handle_bar_str: &str;
if distributed.is_none() {
    handle_bar_str = "filter.rs.handlebars";
} else {
    handle_bar_str = "distributed_filter.rs.handlebars";
}
write_to_handlebars(
    &codegen_object,
    bin_dir.join(handle_bar_str),
    PathBuf::from(matches.value_of("output").unwrap()),
);
```

Reply:
Gotcha.  Should be fixed to that now.